TAScheduler/AcceptanceTests/test_delete_section.py

```diff
@@ -0,0 +1,78 @@
+# PBI #10: Delete Section
+from datetime import datetime
+
+from django.test import TestCase, Client
+
+from TAScheduler.models import Section, Course, Lecture, Lab
+
+
+class SuccessDelete(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.courseList = list()  # holding list of test courses so easier to call later
+        self.secList = list()
+        # Creating 3 Courses: hardcoded "numSec","modality","credits"
+        for i in [1, 2, 3]:
+            testCourse = Course(course_id=i, semester="semester" + str(i), name="name" + str(i), num_of_sections=2,
+                                modality="Remote", credts=3, )
+            testCourse.save()
+            self.courseList = self.courseList.append(testCourse)
+        # Creating 3 Sections: hardcoded "meetingTime"
+        for i in [1, 2, 3]:
+            testSection = Section(section_id=i, course=self.courseList[i - 1], location="location" + str(i),
+                                  meeting_time=datetime(2023, 1, 1, 1, 1, 1))
+            testSection.save()
+            self.secList = self.secList.append(testSection)
+##
+    # 1] Section not found in the section list
+    def test_correctDelete(self):
+        resp = self.client.post("home/managesection/delete", data={"section": self.secList[0]})
+        self.assertNotIn(self.secList[0], Section.objects.all,
+                         msg="The 1st section should no longer be in the database.")
+
+    # 2] Number of sections should have decreased if deleted properly
+    def test_correctNumSec(self):
```

Comment:
Feels unnecessary. If course is deleted and we already tested to see if it was deleted. No need to do it again.

Reply:
You're right. It definitely might be redundant but I figured I would just leave in case some weird replacing is happening in the system. Example: maybe it wants to delete section 1, so it deletes selection 1 and somehow replaces that with something else. It seems pretty unlikely tbh but that was the thought process : P